CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# Contributing Guidelines
+
+### General
+* Keep in mind that this is a design project. Contributions of all kinds (issues, ideas, proposals), not just code, are highly appreciated.
+* Pull requests are welcome with the understanding that changes proposed to actual operations, parameters and responses will be carefully evaluated and discussed, and may not always be accepted.
+* All contributions including documentation, filenames and discussions should be written in English language.
+
+### Issues
+Our issue tracker can be used to report issues and propose changes to the current or next version of the data.world API specification.
+
+Please follow these guidelines before opening an issue:
+
+- Make sure your issue is not a duplicate.
+- Make sure your issue is relevant to the specification.
+# Contribute Code
+
+### Fork the Project
+
+Fork the project [on Github](https://github.com/datadotworld/data.world-py) and check out your copy.
+
+```sh
+$ git clone https://github.com/[YOUR_GITHUB_NAME]/data.world-py.git
+$ cd data.world-py
+$ git remote add upstream https://github.com/datadotworld/data.world-py.git
+```
+
+### Install and Test
+
+Run the command below to install packages required:
+
+```sh
+$ python setup.py install
+```
+
+Run tests:
+
+```sh
+$ python setup.py test
+```
+
+### Updating Swagger Definition
+
+Once new API endpoints/paths are introduced, there is a need to update `data.world-py/datadotworld/client/swagger-dwapi-def.json` file with [data.world APIs swagger.json](https://api.data.world/v0/swagger.json).
+
+So, navigate to [data.world APIs swagger.json](https://api.data.world/v0/swagger.json), copy the and paste the json into the `data.world-py/datadotworld/client/swagger-dwapi-def.json` file in your project folder.
```

Comment:
This process is automated via with make. Should we even describe what the manual process looks like? I'd rather just promote the use of make.